.github/scripts/compile.sh

```diff
@@ -88,6 +96,23 @@ DESTDIR="${LWS_INSTALL_DIR}" cmake --install "${LWS_BUILD_DIR}"
 tar -z -C "${LWS_INSTALL_DIR}" -cf /libwebsockets-image.tar.gz usr
 #endregion libwebsockets
 
+#region libunwind
+pushd "${LIBUNWIND_DIR}"
+autoreconf -i
+./configure
+# This make install will install the libunwind in /usr/local/lib
+# and the include files in /usr/local/include and when skupper-router is compiled
+# in the subsequent steps, it can find the libunwind related files in /usr/local
```

Comment:
The LWS above does exactly the same thing, installing into two places, first place so it can be used when compiling, and second place so that it can be tarred. It would make more sense to place these extensive comments on the first use of this trick (for libwebsocket), not on the second use here.